doc/overview.rst

```diff
@@ -41,11 +31,47 @@ when running on your machine::
     params.sws_ssh = 'kasga'
     params.sws_dir = '/data06/larsoner/sss_work'
 
+.. note:: Head position estimation is now available using
+          ``hp_type='python'``. Automatic bad channel detection like
+          MaxFilter provides will also be available hopefully soon.
+
 Using per-machine config files rather than per-script variables should
 help increase portability of scripts without hurting reproducibility
 (assuming we all use the same version of MaxFilter, which should be a
 safe assumption).
 
+To test that things are configured correctly, you can do:
+
+.. code-block:: console
+
+    $ python -c "import mnefun; mnefun.check_sws()"
```

Comment:
I verified that I could get this message (using a Jupyter notebook for documentation in my case).  I also was able to run the SSS step (set for local processing of tSSS and head position estimation on Kasga) on a subject that had previously been successful using the old way to set the sws parameters.